threedi_api_qgis_client/utils.py

```diff
@@ -119,3 +119,19 @@ def check_download_checksum(download, filename):
         data = file_to_check.read()
         md5_returned = hashlib.md5(data).hexdigest()
         return download.etag == md5_returned
+
+
+def handle_api_exceptions(e):
```

Comment:
Really nice to have such an error-extraction function, btw!

It is probably useful in lots of other places in the code, but that's something for later.

Reply:
I was also thinking about some general function to handle every kind of API exceptions, but that deserves for separate ticket. :)